subsys/bluetooth/controller/Kconfig

```diff
@@ -495,6 +495,15 @@ config BT_CTLR_ADVANCED_FEATURES
 menu "Advanced features"
 	visible if BT_CTLR_ADVANCED_FEATURES
 
+config BT_LL_SW_LLCP_LEGACY
+	bool "Software-based BLE Link Layer"
```

Comment:
Shouldn't the caption be something like "Use Legacy LLCP implementation" as well as an updated Help section?